components/x-teaser-timeline/src/lib/transform.js

```diff
@@ -67,15 +50,27 @@ const addItemGroupTitles = (itemGroups, localTodayDate) => {
 	});
 };
 
-const getItemGroups = ({items, timezoneOffset, localTodayDate, latestItemsTime}) => {
-	if (!items || !Array.isArray(items) || items.length === 0) {
+const getItemGroups = ({sortedItems, timezoneOffset, localTodayDate, latestItemsTime, latestItemsAgeHours}) => {
+	if (!sortedItems || !Array.isArray(sortedItems) || sortedItems.length === 0) {
 		return [];
 	}
 
-	let itemGroups = groupItemsByLocalisedDate(items, timezoneOffset);
+	const isLatestItemTimeWithinRange = latestItemsAgeHours ?
```

Comment:
Might just be me but find this a little tricky to grok, is there a way of making these rules a bit more explicit?

Reply:
@robsquires I've refactored a bit and added a comment. Please have another look.